lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -326,6 +330,23 @@ instance ToText SortOrder where
 instance FromText SortOrder where
     fromText = fromTextToBoundedEnum SnakeLowerCase
 
+defaultTxSortOrder :: SortOrder
+defaultTxSortOrder =  Descending
+
+-- Represents the range [start, end]
+data Filter a = Filter
+    { filterStart :: Maybe a
+    , filterEnd :: Maybe a
+    } deriving (Eq, Show)
```

Comment:
Perhaps call this `Range`? After all, your comment states "Represents the **range**". :)

Other suggestions:

* `noFilter` -> `infiniteRange` or `openRange`.
* `matchesFilter` -> `isWithinRange` or `withinRange`.

Reply:
👍 I went with `Range`, `isWithinRange` and `wholeRange`.

Reply:
I hadn't thought of `wholeRange`. Nice name!